src/Board/BoardDesigner.java

```diff
@@ -36,9 +37,31 @@
      * This method creates tile-objects with the given tile-values from previous method. It always creates two identical tiles to form pairs for the matching game.
      * @return a list with tile-objects.
      */
-    private List<Tile> createTileObjects(int nbTiles) {
+    private List<Tile> createTileObjects(int nbTiles, String difficultyLevel) {
         List<Tile> tilesForBoard = new ArrayList<>();
         List<String> tileValues = readTileValues(nbTiles);
+        Tile tileSkip = new Tile(false,null,"Skip");
+        Tile tileShuffle = new Tile(false,null,"Shuffle");
+        switch(difficultyLevel) {
+            case "B":
+                tilesForBoard.add(tileSkip);
+                tilesForBoard.add(tileShuffle);
+                break;
+            case "C":
+                tilesForBoard.add(tileSkip);
+                tilesForBoard.add(tileSkip);
+                tilesForBoard.add(tileShuffle);
+                tilesForBoard.add(tileShuffle);
+                break;
+            case "D":
+                tilesForBoard.add(tileSkip);
```

Comment:
Het is beter om drie keer een nieuwe skip tegel toe te voegen dan drie keer dezelfde. Enig idee waarom :p?